src/main/resources/templates/email/deliverable/readyForReview/body.txt.ftl

```diff
@@ -4,6 +4,7 @@ Deliverable was submitted by ${participantName}
 A deliverable was submitted by ${participantName} and is ready for review for approval.
 
 ${deliverableUrl}
+${deliverable}
```

Comment:
This should probably include the project, category, and deliverable name too; `${deliverable}` will render the whole object as a string.

Reply:
Oh, ok! Will update it, but also, what is the purpose of rendering the strings in this file? I wan't sure if I needed to include all the new text here

Reply:
It's probably a bit of overengineering, but the idea is that if someone has HTML email disabled or they're getting an excerpt of the email over a channel that doesn't support HTML (phone notification, etc.), they'll see the plain text version instead.